### PR TITLE
[15.0][MIG] barcodes generator product

### PR DESCRIPTION
**Pull request by sla-subteno-it**

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ exclude: |
   (LICENSE.*|COPYING.*)
 default_language_version:
   python: python3
-  node: "14.18.0"
+  node: "17.1.0"
```
Comment (legalsylvain) [inline .pre-commit-config.yaml]: Why this change for that module ?

Comment (sla-subteno-it) [reply .pre-commit-config.yaml]: @legalsylvain I have a Mac M1, the 14.18.0 has not package for M1 so, I used the last one.

Comment (legalsylvain) [reply .pre-commit-config.yaml]: I don't think you can change this setting for a single repo. If a setting is incorrect we should ping OCA maintainers to change it globally.
@pedrobaeza any point of view ?

Comment (pedrobaeza) [reply .pre-commit-config.yaml]: Yes, indeed that's something that should be discussed in oca-addons-repo-template in an issue to make all OS compatible, but changing this now without testing other options may crash other configurations except yours. Please revert it or keep it by yourself. Anyways, I have heard that it's not easy to make Odoo work in a M1.

Comment (sla-subteno-it) [reply .pre-commit-config.yaml]: @legalsylvain @pedrobaeza I just fix it, tI reverted the version.

**Comment by legalsylvain** [comment]
@sla-subteno-it could you add me as maintainers for the both abstract and product modules ? I forget to do it when I developed them.
Thanks.

**Comment by sebastienbeau** [comment]
Hi Seb ! Great to see you here
Maybe this fix can interest you https://github.com/OCA/stock-logistics-barcode/pull/381
